Makefile

```diff
@@ -28,24 +32,30 @@ coveralls: test
 
 ### Code quality and safety commands ###
 
-lint: bandit black mypy pylama safety ## Run linting, code quality, and safety checks
-
-bandit:
-	pipenv run bandit -r ccslips
+# linting commands
+lint: black mypy ruff safety 
 
 black:
 	pipenv run black --check --diff .
 
 mypy:
-	pipenv run mypy ccslips
+	pipenv run mypy .
 
-pylama:
-	pipenv run pylama --options setup.cfg
+ruff:
+	pipenv run ruff check .
 
 safety:
 	pipenv check
 	pipenv verify
 
+# apply changes to resolve any linting errors
+lint-apply: black-apply ruff-apply
+
+black-apply: 
+	pipenv run black .
+
+ruff-apply: 
+	pipenv run ruff check --fix .
 
 ### Terraform-generated Developer Deploy Commands for Dev environment ###
 dist-dev: ## Build docker container (intended for developer-based manual build)
```

Comment:
Hmm, can we delete comments related to Lambda repos if they are irrelevant (e.g., lines 72-74 and 93-95)?

Reply:
Agreed, we should do that going forward unless you have any objection @ghukill ?

Reply:
No objections, think it's a great idea.